eras/babbage/impl/src/Cardano/Ledger/Babbage/Translation.hs

```diff
@@ -180,6 +181,19 @@ translateTxOut (Alonzo.TxOut addr value dh) = TxOut addr value d SNothing
       SNothing -> NoDatum
       SJust d' -> DatumHash d'
 
+-- | A word is 8 bytes, so to convert from coinsPerUTxOWord to coinsPerUTxOByte,
+-- we divide by 8, after first adding 7 so that we are rounding up.
+coinsPerUTxOWordToCoinsPerUTxOByte :: Coin -> Coin
+coinsPerUTxOWordToCoinsPerUTxOByte (Coin c) = Coin $ c + 7 `div` 8
```

Comment:
:wink:  For this to be correct, we need some parenthesis. `div` has a custom fixity defined for it, to match the regular division:
```haskell
λ> :i `div`
type Integral :: * -> Constraint
class (Real a, Enum a) => Integral a where
  ...
  div :: a -> a -> a
  ...
  	-- Defined in ‘GHC.Real’
infixl 7 `div`
λ> :i (/)
type Fractional :: * -> Constraint
class Num a => Fractional a where
  (/) :: a -> a -> a
  ...
  	-- Defined in ‘GHC.Real’
infixl 7 /
λ> :i (+)
type Num :: * -> Constraint
class Num a where
  (+) :: a -> a -> a
  ...
  	-- Defined in ‘GHC.Num’
infixl 6 +
```

So, in your implementation division will happen first. Instead it should be:
```suggestion
coinsPerUTxOWordToCoinsPerUTxOByte (Coin c) = Coin $ (c + 7) `div` 8
```

Reply:
yes indeed, thank you for catching this!